### PR TITLE
[Example] Add a instruction to redirect the user to gluon-cv

### LINKED ISSUE #9622: Unable to reproduce the published mAP for example/ssd with VGGNET model VOC0712  data
I was not able to reproduce the published mAP with SSD VGG16_reduced model  with VOC0712 train-validation and VOC07 test data (300x300 image size) with MxNet (Expected 0.77, Actual 0.73).

I used an effective batch size of 32 on 2 GPUs with a learning rate of 0.001. Rest of the hyper-parameters were as documented in GitHub. 

Trained on AWS Deep Learning AMI Ubuntu Linux - 2.4_Oct2017 (ami-37bb714d) p2.8xlarge

[ssd_caffe_test.txt](https://github.com/apache/incubator-mxnet/files/1675704/ssd_caffe_test.txt)
[ssd_caffe_train.txt](https://github.com/apache/incubator-mxnet/files/1675705/ssd_caffe_train.txt)
[ssd_mxnet_trainval.txt](https://github.com/apache/incubator-mxnet/files/1675706/ssd_mxnet_trainval.txt)

### PR DESCRIPTION
**Pull request by stu1130**
## Description ##
The model cannot achieve the mAP we claimed on latest mxnet version, redirect the user to the gluon-cv
Fix #9622

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Feature1, tests, (and when applicable, API doc)
- [x] Feature2, tests, (and when applicable, API doc)

## Comments ##
N/A

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-merge]
